scripts/killall_sglang.sh

```diff
@@ -1,5 +1,9 @@
 # Kill all SGLang processes and free the GPU memory.
 
+nvidia-smi
 kill -9 $(ps aux | grep 'multiprocessing.spawn' | grep -v 'grep' | awk '{print $2}')
 kill -9 $(ps aux | grep 'sglang.launch_server' | grep -v 'grep' | awk '{print $2}')
 kill -9 $(ps aux | grep 'sglang.bench' | grep -v 'grep' | awk '{print $2}')
+
+# Kill all processes occupying GPU memory, and enable it when needed.
```

Comment:
I sometimes use this command in the local development environment, and it's not enabled by default. Let's leave it as notes.